### DISCUSSION TITLE
DSO Tracking

### DISCUSSION THREAD

**Discussion by JDAstroPhoto** [mworion/MountWizzard4#260] [Q&A] [answered]
I can't seem to figure out how to install SIMBAD for DSO Tracking in MW4.  Can someone please point me in the right direction.
Thanks

**Comment by mworion** [comment, marked as answer]
You don't have to install the database as it's an online database. Please ensure that MW4 is set to online:

![online](https://user-images.githubusercontent.com/6632980/184500560-32894f9f-03ce-4bf2-b0ac-747f65de8c5a.png)

Michel

**Reply by JDAstroPhoto** [reply]
Thank-you very much.  You were correct, I didn't have that checked.  I have it checked now and saved in my configuration.

**Reply by JDAstroPhoto** [reply]
I presume the DSO track is supposed to display in Hemisphere.  I can't get mine to display.  
Points: 40, Duration: 8hrs Time shift: 0
RA: 5:36:20 DEC: 34:8:0 --> M36 which is visible in the sky right now, AZ: 279 and Alt 79
So I would expect there to be a DSO track line of 40 points on my Hemisphere display, but it is blank.
BTW: all the other point generation routines work great, they display the points on Hemisphere.

**Reply by mworion** [reply]
Hi,
I assume that you mount is down and / or not connected to MW4. In this case time data and locations is missing to calculate DSO track. Please connect the mount. 
In next releases this is disabled in such a situation:

![2022-08-14-07-22-20-MountWizzard4_-_v3 0 0a0](https://user-images.githubusercontent.com/6632980/184526776-112bc5a0-b8da-4c1c-9352-4798b65249c9.png)

Michel

**Reply by JDAstroPhoto** [reply]
Thank-you again for your time, you are correct, I did not have the Mount connected.

**Reply by mworion** [reply]
You’re welcome
